src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import SidebarWrapper from '../../styles/SidebarWrapper.styled';
+import SidebarCategory from './SidebarCategory/SidebarCategory';
+
+const Sidebar = ({ categories, activeCategoryFilters, handleCategoryClick }) => {
+    return (
+        <SidebarWrapper>
+            {categories?.results.map((category) => (
+                <SidebarCategory

```

Comment:
It would be better if these categories in the sidebar had the cursor "pointer", so the user realize those buttons are clickable.